acceptancetests/assess_persistent_storage.py

```diff
@@ -144,6 +433,18 @@ def parse_args(argv):
     return parser.parse_args(argv)
 
 
+def assess_persistent_storage(client):
+    # PR7635 landed later today, brings persistent storage to lxd
+    # The persistent storage type on lxd is 'lxd'
+    # https://github.com/juju/juju/pull/7635
+    environment = os.getenv('ENV', default='parallel-aws')
+    if environment == 'parallel-lxd':
+        log.error('TODO: Add persistent storage test on lxd.')
+        sys.exit(1)
```

Comment:
It is intentional, as the filesystem in lxd is different to aws and there is a filesystem type check in the test. When deploying the charm, multiple filesystem type are available to choose but not all of them can be "persistent", and we need to specifically point out which one we're going to use in order to finish the user scenario. Once charm deployed, a filesystem type check is necessary in case Juju use anything other than the one we specified - we had bug on this before.